packages/gestalt/src/SideNavigationTopItem.js

```diff
@@ -89,188 +97,195 @@ type Props = {|
 /**
  * Use [SideNavigation.TopItem](https://gestalt.pinterest.systems/web/sidenavigation#SideNavigation.TopItem) to redirect the user to a different page or section. SideNavigation.TopItem must be used at the top level of SideNavigation. It supports badges, icons, counters, and notifications.
  */
-export default function SideNavigationTopItem({
-  active,
-  href,
-  badge,
-  counter,
-  icon,
-  label,
-  primaryAction,
-  notificationAccessibilityLabel,
-  onClick,
-}: Props): Node {
-  const { nestedLevel } = useNesting();
+const SideNavigationTopItemWithForwardRef: AbstractComponent<Props, HTMLLIElement> = forwardRef(
```

Comment:
Document ref
<img width="607" alt="Screenshot by Dropbox Capture" src="https://github.com/pinterest/gestalt/assets/10593890/38cb8996-0e0b-49c7-b711-bbf12c362951">

Reply:
And also in TS files